dockerfile-image-update/src/main/java/com/salesforce/dockerfileimageupdate/model/FromInstruction.java

```diff
@@ -190,6 +191,17 @@ public boolean hasADifferentTag(String expectedTag) {
         return !tag.trim().equals(expectedTag.trim());
     }
 
+    /**
+     * Determines whether the comment has mentioned "no-dfiu" to ignore creating dfiu PR
+     * If "no-dfiu" present in comment, PR should be ignored
+     */
+    public boolean shouldNotIgnorePR() {
```

Comment:
Can you rename this to ‘ignorePR’ so we don’t have a negation in the function name?

Reply:
Done